README.md

```diff
@@ -14,7 +14,7 @@ Utility for generating buildstamp, a file with build-time info.
 }
 ```
 ## Packages
-### core
+### [core](https://github.com/qiwi/buildstamp/tree/master/packages/core)
```

Comment:
use package names as in npm
[buildstamp](...)
Package with buildstamp CLI and API → Buildstamp generator utility supporting native JS and CLI API

[buildstamp-bin](...)